Modules/SubModule_ASL/xASL_wrp_CreateAnalysisMask.m

```diff
@@ -112,6 +115,20 @@ function xASL_wrp_CreateAnalysisMask(x)
 MaskVascularMNI(pWM>0.9) = 1; % Remove WM vascular spots
 MaskVascularMNI(pCSF>0.9) = 1; % Remove CSF vascular spots
 
+% Obtain brain mask for image processing (e.g., BASIL)
+BrainMaskProcessingStandardSpace = (pGM+pWM+pCSF)>0.1 & FoVim;
+
+%% 3C. Save brain mask for image processing (e.g., BASIL)
+% This mask can be used for fitting data, e.g., BASIL, fitting ATT, Tex,
+% etc. that is not useful in extracranial voxels, and beneficial to speed
+% up by only processing intracranial voxels. On the other hand, it can be
+% helpful to also process some voxels outside the brain for inspecting
+% artifacts, and registration errors shouldn't lead to accidentally
+% excluding intracranial voxels. Therefore, we use a rather inclusive mask.
+
+xASL_io_SaveNifti(x.P.Path_PWI, x.P.Path_BrainMaskProcessing, BrainMaskProcessingNativeSpace, 8, false);
```

Comment:
Just to be sure that I don't overlook anything. This mask is only pGM+pWM+pCSF, right? Cause we want to keep the vascular masks and everything in.

What about tumors and so on. Are we sure that all irregular things and lesions are included as we want to process all these, just exclude outside of the brain.

Reply:
Yes, you're right. I have now explained all this in the header, is this to your liking?